apps/src/templates/teacherDashboard/TeacherDashboardHeader.jsx

```diff
@@ -26,12 +84,65 @@ export default class TeacherDashboardHeader extends React.Component {
           isRtl={true}
           chevronSide="left"
         />
-        <div style={styles.headerContainer}>
-          <h1>{this.props.sectionName}</h1>
-          <SelectSectionDropdown />
+        <div style={styles.header}>
+          <div>
+            <h1>{this.selectedSection.name}</h1>
+            <div>
+              <span style={styles.sectionPrompt}>
+                {i18n.assignedToWithColon()}{' '}
+              </span>
+              {this.props.selectedSectionScript.name}
+            </div>
+          </div>
+          <div style={styles.rightColumn}>
+            <div style={styles.buttonSection}>
+              <Button
+                __useDeprecatedTag
+                onClick={() => {
+                  this.props.openEditSectionDialog(this.selectedSection.id);
+                  recordOpenEditSectionDetails(
+                    this.selectedSection.id,
+                    'dashboard_header'
+                  );
+                }}
+                icon="gear"
+                size="narrow"
+                color="gray"
```

Comment:
Can we use the Button component's specified gray just in case we decide to change that default down the road? Like this: https://github.com/code-dot-org/code-dot-org/blob/8d8904a48c56f37793a87de603c9fa313a6f04bb/apps/src/templates/teacherDashboard/DialogFooter.story.jsx#L32

Reply:
I will implement this in a follow up PR! trying to get this merged asap right now, ty for the pointer